packages/internet-header/src/components/post-internet-footer/components/post-footer-block-contact.component.tsx

```diff
@@ -1,8 +1,10 @@
 import { h } from '@stencil/core';
 import { BlockEntity } from '../../../models/footer.model';
 
-const getContentHours = (hours: string) => hours.replace(/<[^>]*>?/gm, '');
-
+function stripHtml(html: string): string {
+  const doc = new DOMParser().parseFromString(html, 'text/html');
```

Comment:
DOMParser().parseFromString is not executing any script. It can still contain malicious code like onclick event that will work if it's not santitize inside the DOMParser and that the result is injected bad in the current DOM document. In our case, it's safe because using "textContent" will always allow only text. Also, this technic is used by [DOMPurify library](https://github.com/cure53/DOMPurify/tree/main). 

More infos:
* [Discussion about if this technic is safe or not in web-dev repository](https://github.com/GoogleChrome/web.dev/issues/6890)
* [Example of how to sanitize HTML with this technic](https://gomakethings.com/how-to-sanitize-html-strings-with-vanilla-js-to-reduce-your-risk-of-xss-attacks/)